tests/conftest.py

```diff
@@ -35,6 +42,20 @@ def celery_app(mocker):
     ctx.pop()
 
 
+@pytest.fixture
+def app_with_mocked_logger(mocker, tmpdir):
+    """Patch `create_logger` to return a mock logger that is made accessible on `app.logger`"""
+    mocker.patch(
+        "flask.sansio.app.create_logger",
```

Comment:
oops flask needs to be above version 3.0 to mock this - i'll add a PR to fix this

Reply:
Sorry missed this.. hmm yes there's a broader problem with dependency versions, libraries and tests. But that isn't a problem unique to our project.